docs/Xamarin.Forms.Core/index.xml

```diff
@@ -394,10 +419,15 @@
     </Namespace>
     <Namespace Name="Xamarin.Forms.Internals">
       <Type Name="DynamicResource" Kind="Class" />
+      <Type Name="EvalRequested" Kind="Class" />
       <Type Name="IDataTemplate" Kind="Interface" />
       <Type Name="IDynamicResourceHandler" Kind="Interface" />
       <Type Name="INameScope" Kind="Interface" />
+      <Type Name="InvalidationTrigger" Kind="Enumeration" />
       <Type Name="NameScope" Kind="Class" />
+      <Type Name="NavigationRequestedEventArgs" Kind="Class" />
```

Comment:
There's a couple of extra stuff in here on (index.xml), but i know what it is and why it's missing, i do think @hartez had a pr to fix this, so no issue here.